lib/appium_console.rb

```diff
@@ -9,9 +9,17 @@ def self.warn args
 
 require 'pry'
 
-Pry.send(:define_singleton_method, :pry_load_appium_txt) do |opts={}|
-  verbose = opts.fetch :verbose, false
-  Appium.load_appium_txt file: Dir.pwd + '/appium.txt', verbose: verbose
+Pry.send(:define_singleton_method, :pry_load_appium_txt) do |opts = {}|
+  if ARGV.size >= 2
+    command       = ARGV[0].to_sym
+    option        = ARGV[1]
+    opts[command] = option
+  end
+
+  verbose       = opts.fetch :verbose, false
+  dir           = opts.fetch :appium_dir, Dir.pwd
+  path          = File.join dir, "/appium.txt"
```

Comment:
My mistake... Overall, I must say I'm not really convinced about the current arg management stuff. I saw issue #43 and I might be able to give it a try next week and add tests to it so this pr will become irrelevant. I'd rather do something clean and refactor stuff along the way than have this "thing" merged. I'll keep you posted.